community/working-groups.md

```diff
@@ -0,0 +1,57 @@
+## Working Groups
+
+A working group is a group of people that are interested in working on a common
+topic. Working groups are informal and have no official authority. All working
+groups have a public channel on the [Julia Slack][slack]. Many working groups
+also hold regular meetings that are held online and are open to anyone. Meeting
+links are posted in the relevant Slack channels.
+
+The working group meetings are also listed on the
+[community events calendar][commmunity-events-calendar].
+
+### List of working groups
+
+Note: "ET" refers to Eastern Time.
+
+| Working Group                    | Slack channel       | Meetings                                 | Contact                                                         |
+| -------------------------------- | ------------------- | ---------------------------------------- | --------------------------------------------------------------- |
+| Base Julia CI                    | `#ci-dev`           | Every other Tuesday at 2:15pm ET         | Elliot Saba (@staticfloat), Dilum Aluthge (@DilumAluthge)       |
```

Comment:
I gather these are sorted alphabetically, so I'm going to add "Astronomy" first
```suggestion
| Astronomy | `#astro` | Fourth Friday of each month at 19:00 UTC | [@giordano](https://github.com/giordano), [@mileslucas](https://github.com/mileslucas)|
| Base Julia CI                    | `#ci-dev`           | Every other Tuesday at 2:15pm ET         | Elliot Saba (@staticfloat), Dilum Aluthge (@DilumAluthge)       |
```

Reply:
Yep, they're ordered alphabetically, because I figured that was the easiest way to order them.